src/components/EventCard.tsx

```diff
@@ -42,11 +42,11 @@ const EventCard = ({
     <div className="flex my-8 md:my-10">
       <EventDate startTime={startTime} color={color} />
       <div className="flex-1 px-6 md:px-10">
-        <span className="font-sans">
+        <span className="font-sans md:text-base">
```

Comment:
Nit: Did we want the text size to get smaller on larger breakpoints? I think for desktop screens since there's a lot of space it'd be fine to keep the size as is and not change it

Reply:
Ok! Let's keep it the same size then.

Reply:
I'm such a fool...text-body-mobile already uses text-base...they're already the same size 😭